### PR TITLE
integration-tests: add verbose flag

### PR DESCRIPTION
**Pull request by fgimenez**
With this verbose flag all commands sent through the `integration-tests/testutils/cli` package functions won't print the given command nor its output. This is the default behaviour, can be changed to the previous verbose mode by passing `-v` to the integration-tests execution command.

### CODE REVIEW

**Comment by fgimenez** [comment]
retest this please

**Comment by elopio** [comment]
:+1: 
as we discussed during our today's meeting, once we have better handling of the streams from each tests, we might change the default to be more verbose.

File: integration-tests/testutils/config/config_test.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func testConfigStruct(fileName string) *Config {
 	return &Config{
 		fileName,
 		"testrelease", "testchannel",
-		true, true, true, true}
+		true, true, true, true, true}
```
Comment (mvo5) [inline integration-tests/testutils/config/config_test.go]: Totally unrelated to this actual branch, sorry for this: could we use the field names in the struct init to make it easier to see what each value means? Something like:
```
return &Config{FileName: fileName, Release: "testrelease", Channel: "testchannel", RemoteTestbed: true, ...
```

File: integration-tests/testutils/autopkgtest/autopkgtest.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ func (a *AutoPkgTest) adtRun(testbedOptions string) (err error) {
 	prepareTargetDir(outputDir)
 
 	cmd := []string{
-		"adt-run", "-B",
+		"adt-run", "-B", "-q",
```
Comment (mvo5) [inline integration-tests/testutils/autopkgtest/autopkgtest.go]: Should this `-q` be conditional on the verbose setting in the config? I.e. not setting it when the config is set to verbose? Or is it generally just noise that is added is added when we not set the `-q`.

**Comment by mvo5** [comment]
This looks good, thanks a bunch of making the logs easier to read. I will request a retest, for some reason I can not see the logs of the autopkg run. But I'm keen to see what the new logging looks like :)

**Comment by mvo5** [comment]
retest this please

File: integration-tests/testutils/config/config_test.go (hunk 1 above)
Comment (fgimenez) [inline integration-tests/testutils/config/config_test.go]: Indeed, done

File: integration-tests/testutils/autopkgtest/autopkgtest.go (hunk 2 above)
Comment (fgimenez) [inline integration-tests/testutils/autopkgtest/autopkgtest.go]: Totally, I've added a `Verbose` field

**Comment by fgimenez** [comment]
@mvo5 thanks a lot for your review! :) I've addressed the issues, let me know if anything is missing

**Pull request merged**